src/rviz/robot/robot_link.cpp

```diff
@@ -318,6 +319,26 @@ RobotLink::~RobotLink()
   delete axes_;
   delete details_;
   delete link_property_;
+  errors.erase(this);
+}
+
+void RobotLink::addError(const char* format, ...)
```

Comment:
I was referring to this variadic function that could be simplified by using `QString` or `QStringList`